src/item.cpp

```diff
@@ -737,10 +777,15 @@ bool item::is_sided() const
     return t ? t->sided : false;
 }
 
+namespace
+{
+static const std::string lateral{ "lateral" };
+}
+
 side item::get_side() const
 {
     // MSVC complains if directly cast double to enum
-    return static_cast<side>( static_cast<int>( get_var( "lateral",
+    return static_cast<side>( static_cast<int>( get_var( lateral,
```

Comment:
I agree with `static const std::array`s above, but this look too verbose. Perhaps replace it with `STATIC (std::string("lateral") )` ?

Reply:
I did that and the static initializer code still accounted for a nontrivial number of samples. I'll pull that data for the record because it's interesting, but I'll switch back to STATIC() because in the end it is ultimately a pretty negligible amount of time spent.

Reply:
I understand your desire to squeeze every last bit of performance from this method, but at some point it becomes a tradeoff between performance and readability.